docs/project/changelog.md

```diff
@@ -174,6 +174,9 @@ async function runPythonAsync(code, globals) {
 - {{ Enhancement }} upgraded numpy to 1.21.4
   {pr}`1934`
 
+- {{ Enhancement }} Upgraded scipy to 1.6.1
```

Comment:
Great work! nit: I think you meant 1.7.3 here ?

Reply:
Well I wrote the entry before I switched to 1.7.3. Thanks for pointing it out.